packages/react/src/VideoPlayer/VideoPlayer.stories.module.css

```
@@ -0,0 +1,3 @@
+.customPlayIcon {
+  color: white;
```

I know it's just a story, but to avoid triggering any linters in future...

```suggestion
  color: var(--base-color-scale-white-0);
```